### DISCUSSION TITLE
Displaying the hypothetical plastic slip band of a given slip system

### DISCUSSION THREAD

**Discussion by bibiboom89** [mtex-toolbox/mtex#2212] [Ask Anything]
Hi,

In the tutorial "Crystal Shapes" (https://mtex-toolbox.github.io/CrystalShapes.html):
- It is shown how to display the crystal shape and the slip plane and direction of a given slip system.
- It is also shown how to display over an ebsd map the crystal shapes accordingly to the grains orientations.
- But how can I do both ? Also display the slip plane and slip direction over the ebsd map accordingly to the grains orientations ?

My goal is to know in which direction would be the plastic slip band of each slip system of each grain and making it visible by plotting lines over the grain. Something similar to this:
![image](https://github.com/user-attachments/assets/e7f111df-91e5-46ea-8873-68ef636f5ab5)

I thank you in advance,
Regards,
Bibi

**Comment by Tijmenvermeij** [comment]
In a separate section of the documentation, there is an extensive example of how to plot slip plane traces in grains: https://mtex-toolbox.github.io/SchmidtFactor.html

**Reply by bibiboom89** [reply]
Oh yeah, I haven't found that example yet. 
Thank you, this is exactly what I wanted :)

How can I trace several slip traces for the same grain, all spread out from each other by a given distance d ?  
Similar to my example above.
I guess I would need to modify grains positions, but I can't directly do grains.x = grains.x + d.
